addons/portal_rating/static/src/js/portal_rating_composer.js

```diff
@@ -83,6 +83,7 @@ const RatingPopupComposer = publicWidget.Widget.extend({
         }
 
         // Instantiate the "Portal Composer" widget and insert it into the modal
+        this.options.send_button_label = this.options.default_message_id ? _t("Update review") : _t("Post review");
```

Comment:
The "default_message_id" seems to be set directly when initializing the component, so this piece of code can go inside the "RatingPopupComposer" "willStart" function, right?

Could be directly embedded when defining the options.